requirements-dev.in

```diff
@@ -0,0 +1,9 @@
+-r requirements.txt
+
+flake8==3.7.7
+hypothesis==4.48.0
+jsonschema==3.0.0
+deepmerge==0.1.0
+mock==2.0.0
+pytest==4.6.3
+PyYAML==5.4
```

Comment:
These probably don't need to be pinned to specific versions, requirements-dev.txt takes care of that.